Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -613,6 +613,10 @@ def audit_homepage
       problem "The homepage should start with http or https (URL is #{homepage})."
     end
 
+    unless homepage =~ %r{^(.+)://(.*)/(.*)}
+      problem "#{homepage} must have a slash after the domain name part."
```

Comment:
Since this only applies to URLs without a path component, I think the message could be a bit simpler, like `#{homepage} has to end with a slash.`